### ISSUE TITLE
Snapshot #1507 not working correctly (paperUI, exceptions in logs)

### ISSUE THREAD

**Issue by FlorianSW** [openhab/openhab-core#474]
With the latest snapshot build (1507 iirc), openHAB does not seem to work fine anymore. When I start it, I get the following error messages in the log:

```
2019-01-23 19:34:33.910 [ERROR] [org.eclipse.smarthome.core          ] - bundle org.eclipse.smarthome.core:0.11.0.201901181547 (102)[org.eclipse.smarthome.core.internal.scheduler.PeriodicSchedulerImpl(57)] : doFindMethod: Suitable but non-accessible method found in class org.eclipse.smarthome.core.internal.scheduler.PeriodicSchedulerImpl
2019-01-23 19:34:33.911 [ERROR] [org.eclipse.smarthome.core          ] - bundle org.eclipse.smarthome.core:0.11.0.201901181547 (102)[org.eclipse.smarthome.core.internal.scheduler.PeriodicSchedulerImpl(57)] : findMethod: Suitable but non-accessible method setScheduler found in class org.eclipse.smarthome.core.internal.scheduler.PeriodicSchedulerImpl, subclass of org.eclipse.smarthome.core.internal.scheduler.PeriodicSchedulerImpl
2019-01-23 19:34:33.981 [ERROR] [org.eclipse.smarthome.core          ] - bundle org.eclipse.smarthome.core:0.11.0.201901181547 (102)[org.eclipse.smarthome.core.internal.scheduler.PeriodicSchedulerImpl(57)] : bind method [setScheduler] not found; Component will fail
2019-01-23 19:34:33.983 [ERROR] [org.eclipse.smarthome.core          ] - bundle org.eclipse.smarthome.core:0.11.0.201901181547 (102)[org.eclipse.smarthome.core.internal.scheduler.PeriodicSchedulerImpl(57)] : bind method [setScheduler] not found
2019-01-23 19:34:33.984 [ERROR] [org.eclipse.smarthome.core          ] - bundle org.eclipse.smarthome.core:0.11.0.201901181547 (102)[org.eclipse.smarthome.core.internal.scheduler.PeriodicSchedulerImpl(57)] : doFindMethod: Suitable but non-accessible method found in class org.eclipse.smarthome.core.internal.scheduler.PeriodicSchedulerImpl
2019-01-23 19:34:33.985 [ERROR] [org.eclipse.smarthome.core          ] - bundle org.eclipse.smarthome.core:0.11.0.201901181547 (102)[org.eclipse.smarthome.core.internal.scheduler.PeriodicSchedulerImpl(57)] : findMethod: Suitable but non-accessible method unsetScheduler found in class org.eclipse.smarthome.core.internal.scheduler.PeriodicSchedulerImpl, subclass of org.eclipse.smarthome.core.internal.scheduler.PeriodicSchedulerImpl
2019-01-23 19:34:33.986 [ERROR] [org.eclipse.smarthome.core          ] - bundle org.eclipse.smarthome.core:0.11.0.201901181547 (102)[org.eclipse.smarthome.core.internal.scheduler.PeriodicSchedulerImpl(57)] : unbind method [unsetScheduler] not found; Component will fail
2019-01-23 19:34:33.987 [ERROR] [org.eclipse.smarthome.core          ] - bundle org.eclipse.smarthome.core:0.11.0.201901181547 (102)[org.eclipse.smarthome.core.internal.scheduler.PeriodicSchedulerImpl(57)] : unbind method [unsetScheduler] not found
```
There're also exceptions (no idea when they appear) like:
```
javax.servlet.ServletException: javax.servlet.ServletException: A MultiException has 1 exceptions.  They are:
1. java.lang.IllegalStateException: ServiceLocatorImpl(__HK2_Generated_0,1,433218621) has been shut down

        at org.ops4j.pax.web.service.jetty.internal.JettyServerHandlerCollection.handle(JettyServerHandlerCollection.java:88) ~[?:?]
        at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:132) ~[84:org.eclipse.jetty.server:9.4.12.v20180830]
        at org.eclipse.jetty.server.Server.handle(Server.java:503) ~[84:org.eclipse.jetty.server:9.4.12.v20180830]
        at org.eclipse.jetty.server.HttpChannel.handle(HttpChannel.java:364) [84:org.eclipse.jetty.server:9.4.12.v20180830]
        at org.eclipse.jetty.server.HttpConnection.onFillable(HttpConnection.java:260) [84:org.eclipse.jetty.server:9.4.12.v20180830]
        at org.eclipse.jetty.io.AbstractConnection$ReadCallback.succeeded(AbstractConnection.java:305) [75:org.eclipse.jetty.io:9.4.12.v20180830]
        at org.eclipse.jetty.io.FillInterest.fillable(FillInterest.java:103) [75:org.eclipse.jetty.io:9.4.12.v20180830]
        at org.eclipse.jetty.io.ssl.SslConnection$DecryptedEndPoint.onFillable(SslConnection.java:411) [75:org.eclipse.jetty.io:9.4.12.v20180830]
        at org.eclipse.jetty.io.ssl.SslConnection.onFillable(SslConnection.java:305) [75:org.eclipse.jetty.io:9.4.12.v20180830]
        at org.eclipse.jetty.io.ssl.SslConnection$2.succeeded(SslConnection.java:159) [75:org.eclipse.jetty.io:9.4.12.v20180830]
        at org.eclipse.jetty.io.FillInterest.fillable(FillInterest.java:103) [75:org.eclipse.jetty.io:9.4.12.v20180830]
        at org.eclipse.jetty.io.ChannelEndPoint$2.run(ChannelEndPoint.java:118) [75:org.eclipse.jetty.io:9.4.12.v20180830]
        at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.runTask(EatWhatYouKill.java:333) [87:org.eclipse.jetty.util:9.4.12.v20180830]
        at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.doProduce(EatWhatYouKill.java:310) [87:org.eclipse.jetty.util:9.4.12.v20180830]
        at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.tryProduce(EatWhatYouKill.java:168) [87:org.eclipse.jetty.util:9.4.12.v20180830]
        at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.run(EatWhatYouKill.java:126) [87:org.eclipse.jetty.util:9.4.12.v20180830]
        at org.eclipse.jetty.util.thread.ReservedThreadExecutor$ReservedThread.run(ReservedThreadExecutor.java:366) [87:org.eclipse.jetty.util:9.4.12.v20180830]
        at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:765) [87:org.eclipse.jetty.util:9.4.12.v20180830]
        at org.eclipse.jetty.util.thread.QueuedThreadPool$2.run(QueuedThreadPool.java:683) [87:org.eclipse.jetty.util:9.4.12.v20180830]
        at java.lang.Thread.run(Thread.java:748) [?:?]
```
Also: the PaperUI looks like the one from ESH, not from openHAB. Also, some items does not trigger rules anymore, there're, however, no entries in the log file.

**Comment by FlorianSW** [comment]
It's, btw., the latest docker build on Alpine.

**Comment by Hilbrand** [comment]
First error was already reported in: https://github.com/openhab/openhab-core/issues/473

**Comment by kaikreuzer** [comment]
Paper UI should be fixed already, the logs are reported as #473, so let's close this as a duplicate.
